test/witan/gateway/integration/base.clj

```diff
@@ -19,7 +19,7 @@
   ([p tries]
    (wait-for-pred p tries (or (env :wait-ms) 500)))
   ([p tries ms]
-   (log)
+   (log/info "Waiting for predicate -" tries "x" ms)
```

Comment:
I want to print this because it's not obvious to me when the wait begins after all the Kafka noise